src/api/routers/participants/participantsCreation.ts

```diff
@@ -16,68 +16,61 @@ import { getKcAdminClient } from '../../keycloakAdminClient';
 import { addSite, getSiteList, setSiteClientTypes } from '../../services/adminServiceClient';
 import {
   mapClientTypesToAdminEnums,
-  SiteCreationRequest,
+  SiteCreationRequest
 } from '../../services/adminServiceHelpers';
 import {
   constructAuditTrailObject,
-  performAsyncOperationWithAuditTrail,
+  performAsyncOperationWithAuditTrail
 } from '../../services/auditTrailService';
 import {
-  assignApiParticipantMemberRole,
-  createNewUser,
-  sendInviteEmailToNewUser,
+  doesUserExistInKeycloak
 } from '../../services/kcUsersService';
 import {
   getParticipantTypesByIds,
   ParticipantRequest,
-  sendNewParticipantEmail,
+  sendNewParticipantEmail
 } from '../../services/participantsService';
-import { findUserByEmail } from '../../services/usersService';
+import {
+  createAndInviteKeycloakUser,
+  findUserByEmail,
+  sendInviteEmailToExistingUser
+} from '../../services/usersService';
 import {
   ParticipantCreationAndApprovalPartial,
-  ParticipantCreationRequest,
+  ParticipantCreationRequest
 } from './participantClasses';
 
 export async function validateParticipantCreationRequest(
   participantRequest: z.infer<typeof ParticipantCreationRequest>
 ) {
-  let errorMessage = null;
   const existingParticipant = await Participant.query().findOne(
     'name',
     participantRequest.participantName
   );
   if (existingParticipant) {
-    errorMessage = 'Duplicate participant name';
```

Comment:
These two cases are now valid, so should not error

Reply:
If this isn't an error message anymore, are we doing anything with the strings that get returned (and does the function care or use the different strings being returned)?  Should it just be a boolean function instead?

Reply:
the method still returns an error message if there's an error or `null` if there is no error, so the functionality is still the same as before. If we change it to return a `boolean` then we'd lose the error message. I'm inclined to leave as-is unless you feel strongly? Perhaps my inital change/comment was confusing.